lib/src/Uncertainty/Algorithm/Analytical/openturns/Analytical.hxx

```diff
@@ -57,8 +57,12 @@ public:
   Analytical(const OptimizationAlgorithm & solver,
              const RandomVector & compositeEvent,
              const Point & physicalStartingPoint);
-
-
+             
+  Analytical(const OptimizationAlgorithm & solver,
+             const RandomVector & compositeEvent,
+             const Point & physicalStartingPoint,
+             const Scalar & limitStateTolerance);
```

Comment:
we dont want to add new ctors each time parameters are added, instead it can be already set with the accessor you added

Reply:
So you recommend just to access it by setLimitStateTolerance ?

Reply:
yes